db/schema.rb

```diff
@@ -70,6 +70,15 @@
   add_index "deploy_groups_stages", ["deploy_group_id"], name: "index_deploy_groups_stages_on_deploy_group_id", using: :btree
   add_index "deploy_groups_stages", ["stage_id"], name: "index_deploy_groups_stages_on_stage_id", using: :btree
 
+  create_table "deploy_response_urls", force: :cascade do |t|
+    t.integer  "deploy_id",    limit: 4,   null: false
+    t.string   "response_url", limit: 255, null: false
+    t.datetime "created_at"
```

Comment:
These were generated by `t:timestamps` in [the migration](https://github.com/zendesk/samson/blob/ben/slash-command/plugins/slack_app/db/migrate/20160711222417_create_deploy_response_url_table.rb#L7). Is it standard to override this once it's generated?